app/src/main/res/values/strings.xml

```diff
@@ -28,7 +28,7 @@
 
   <!--Business id Identifier -->
   <string name="identifiertype_bp_passport">BP Passport</string>
-  <string name="identifiertype_bangladesh_national_id">National ID number</string>
```

Comment:
I'm wondering if this should just be a "Heath ID" label instead of national ID. Not necessary to wait for this to merge the PR, but might be worth opening that conversation with the design folks.

Reply:
We will provide different strings based on the countries. Discussed with designers [here](https://simpledotorg.slack.com/archives/CFK2PHJ1L/p1619593931412000?thread_ts=1619588985.411500&cid=CFK2PHJ1L)